diff --git a/CMakeLists.txt b/CMakeLists.txt
index 84846f9..e427288 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -174,8 +174,8 @@ FetchContent_Declare(
     vectorclass
     GIT_REPOSITORY https://github.com/vectorclass/version2
     GIT_TAG master
-    DOWNLOAD_DIR ${PROJECT_SOURCE_DIR}/thirdparty/vcl
-    SOURCE_DIR ${PROJECT_SOURCE_DIR}/thirdparty/vcl
+    DOWNLOAD_DIR ${PROJECT_SOURCE_DIR}/thirdparty/vectorclass
+    SOURCE_DIR ${PROJECT_SOURCE_DIR}/thirdparty/vectorclass
 )
 
 FetchContent_Declare(
@@ -197,10 +197,10 @@ if(NOT vstat_POPULATED)
 endif()
 
 # populate content
-FetchContent_MakeAvailable(xxhash csv nanobench robin_hood pdqsort span-lite fast_float vectorclass)
+FetchContent_MakeAvailable(xxhash csv nanobench robin_hood pdqsort span-lite fast_float vectorclass vstat)
 
 set(THIRDPARTY_INCLUDE_DIRS
-    ${PRATT_PARSER_INCLUDE_DIRS}
+    ${pratt-parser_SOURCE_DIR}/include/pratt-parser
     ${PROJECT_SOURCE_DIR}/thirdparty/CSV/include
     ${PROJECT_SOURCE_DIR}/thirdparty/pdqsort
     ${Taskflow_INCLUDE_DIR}
@@ -208,9 +208,10 @@ set(THIRDPARTY_INCLUDE_DIRS
     ${gsl_SOURCE_DIR}/include
     ${nanobench_SOURCE_DIR}
     ${span-lite_SOURCE_DIR}
-    ${vectorclass_SOURCE_DIR}
+    ${vectorclass_SOURCE_DIR}/../
     ${xxhash_SOURCE_DIR}
-    ${VSTAT_INCLUDE_DIRS}
+    ${vstat_SOURCE_DIR}/include/vstat
+    ${robin_hood_SOURCE_DIR}
 )
 
 # populate optional content
